extensions/amp-auto-ads/0.1/amp-auto-ads.js

```diff
@@ -31,12 +33,14 @@ export class AmpAutoAds extends AMP.BaseElement {
     const type = this.element.getAttribute('type');
     user().assert(type, 'Missing type attribute');
 
-    const configUrl = getConfigUrl(type, this.element);
-    if (!configUrl) {
+    const adNetwork = getAdNetworkConfig(type, this.element);
+    if (!adNetwork) {
       return;
```

Comment:
We need to throw here to blacklist the element (optimization for Resources, no longer have to check this element for anything). We can likely move [this error](https://github.com/ampproject/amphtml/pull/6443/files#diff-e5a0eab686b905efd9c175e2fb05dc01R54) to here, and change it to an assert.